src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/raw/RawAtlasIntegrationTest.java

```diff
@@ -287,7 +287,15 @@ public void testSectioningFromShard()
         Assert.assertEquals(5126, finalAtlas.numberOfAreas());
         Assert.assertEquals(184, finalAtlas.numberOfPoints());
         Assert.assertEquals(271, finalAtlas.numberOfLines());
-        Assert.assertEquals(14, finalAtlas.numberOfRelations());
+
+        /*
+         * This has been updated to 16 instead of 14. This is because two of the relations in
```

Comment:
Not sure this comment is necessary. The check here was on the data, simply changing the number is fine. Thanks for the detailed explanation!

Reply:
I honestly left it just for the purposes of explaining the change in the PR, so you guys didn't think I was just cheating to make a test pass :)

It could have been removed but we merged so fast that I didn't get a chance.